doc/source/notebooks/understanding/models.ipynb

```diff
@@ -24,7 +24,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 27,
```

Comment:
Would be great if you could "Restart&Run all" the notebook to make sure it all runs correctly & so the cells are numbered in a nice sequence!

Reply:
done